plugins/outputs/opentsdb/opentsdb_http.go

```diff
@@ -16,7 +16,7 @@ import (
 type HttpMetric struct {
 	Metric    string            `json:"metric"`
 	Timestamp int64             `json:"timestamp"`
-	Value     string            `json:"value"`
+	Value     interface{}       `json:"value"`
```

Comment:
@sparrc The OpenTSDB API is a bit loose. It says it doesn't accept string values but it does accept numbers quoted or not. So that is why it is working. Although the API was returning errors for string(characters) values it did process all metrics compared to the telnet API which just bail out at the first error. When you use tools between telegraf and OpenTSDB and they are more strict, it can cause problems so this is why I am making the change.